### ISSUE TITLE
Turris Omnia

### ISSUE THREAD

**Issue by stokito** [josedelinux/openwrt-jdk#1]
I tried your archive on the Turris Omnia router and got the error:
```
Error relocating /usr/lib/jvm/lib/aarch32/jli/libjli.so: __dlsym_time64: symbol not found
Error relocating /usr/lib/jvm/lib/aarch32/jli/libjli.so: __gettimeofday_time64: symbol not found
Error relocating /usr/lib/jvm/lib/aarch32/jli/libjli.so: __stat_time64: symbol not found
```

Alpine v3.13 switched to musl v1.2.0
https://wiki.alpinelinux.org/wiki/Release_Notes_for_Alpine_3.13.0

But the TurrisOS have an older version 1.1. And explanation of the bug https://stackoverflow.com/a/62542720/1049542

The OpenWrt 23 also has a newer musl v1.2.4. So on the vanilla OpenWrt you shouldn't have the problem.

You can check the musl version with ldd:
```
$ ldd
musl libc (armhf)
Version 1.1.24
```

Instead I modified the original script and installed the OpenJDK8 successfully.
See my gist https://gist.github.com/stokito/7dd425da5a12abce8b39dda1bd1106d7

Your repository just untar and tar back the openjdk from Alpine. It's better and safer to install directly from the Alpine repo.

Can we change your repo to just a one script that downloads and install the OpenJDK? And remove those archives to not confuse people.

Then we can have issues and accept PR from others.

**Comment by josedelinux** [comment]
Hi stokito,

Thanks for reporting this issue. I've added statements for clarification in the README file, as you suggested.

I really appreciate your help in pointing out mistakes in my work on this issue. Your contribution is always welcome. I'm also open to any other feedback you have. Feel free to change the script code or README as you like.

Specifically, you mentioned that the script should download and install OpenJDK directly from the Alpine repository. In my humble opinion, Building binaries from source code is the best solution, the second best is repackaging them as opkg files.

But I'm busy with other projects right now, so I don't have a lot of time to refine them. But if I do, I'll try to become a package maintainer.

I've also updated the README file to reflect the changes.

I'm still learning, and I'm always looking for ways to improve my skills. Likewise, I would really appreciate your feedback on the changes I've made.

Thanks again for your help!

Stay strong Ukraine, Слава Україні!

Best regards,
Jose
